FrontEnd/src/main/java/resourceDisplay/IndexController.java

```diff
@@ -24,6 +24,25 @@
     private static Spanner spanner;
     private static DatabaseId db;
     private static DatabaseClient dbClient;
+    private static String userID;
+    public static String workspaceID = "--";
```

Comment:
why public memeber?

Reply:
Thought we needed this to be public for the template but we don't. fixed it!